docs/reference/content/getting-started/quick-tour.md

```diff
@@ -303,6 +301,53 @@ myDoc = collection.find().projection(excludeId()).first();
 System.out.println(myDoc.toJson());
 ```
 
+## Updating documents
+
+There are numerous [update operators](http://docs.mongodb.org/manual/reference/operator/update-field/)
+supported by MongoDB.
+
+If you want to update a single document, the [`updateOne`]({{< apiref "com/mongodb/client/MongoCollection.html#updateOne-org.bson.conversions.Bson-org.bson.conversions.Bson-">}})
```

Comment:
How about:
    To update a single document, use the updateOne methods to specify a filter and the update document.